login-workflow/src/screens/EulaScreen/EulaScreen.tsx

```diff
@@ -17,13 +17,14 @@ export const EulaScreen: React.FC<EulaScreenProps> = (props) => {
         onEulaAcceptedChange = (accepted: boolean): boolean => accepted,
         eulaContent,
         checkboxLabel = t('bluiRegistration:REGISTRATION.EULA.AGREE_TERMS'),
+        checkboxProps,
+        htmlEula,
         initialCheckboxValue,
         errorDisplayConfig = errorManagerConfig,
     } = props;
-
-    const [eulaAccepted, setEulaAccepted] = useState(screenData.Eula.accepted ?? initialCheckboxValue);
+    const [eulaAccepted, setEulaAccepted] = useState(initialCheckboxValue ?? screenData.Eula.accepted);
     const [isLoading, setIsLoading] = useState(true);
-    const [eulaData, setEulaData] = useState<string>();
+    const [eulaData, setEulaData] = useState<any>();
```

Comment:
Instead of any, we can give `string | JSX.Element`

Reply:
Or react node?

Reply:
Updated with `string | JSX.Element`